<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ib_Login</class>
 <widget class="QWidget" name="ib_Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="main_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#main_widget {
	border-image:url(&quot;:/Images/login_bg.jpg&quot;);
}</string>
   </property>
   <widget class="QWidget" name="login_frame" native="true">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>150</y>
      <width>350</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#login_frame {
	background-color: rgba(255, 255, 255, 60%);
	border-radius: 20px;
	border-style:solid;
	border-width:2px;
	border-color:rgb(213, 213, 213);
}</string>
    </property>
    <widget class="QLabel" name="lb_head">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>10</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>智慧公交</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lb_subtitle">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>60</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>公交调度排版系统</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="login_name" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="lb_uname">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>0</y>
        <width>30</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_uname">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>5</y>
        <width>180</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="login_password" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
     <widget class="QLabel" name="lb_password">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>0</y>
        <width>30</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_password">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>5</y>
        <width>180</width>
        <height>30</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btn_login">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>220</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_clear">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>220</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>le_uname</tabstop>
  <tabstop>le_password</tabstop>
  <tabstop>btn_clear</tabstop>
  <tabstop>btn_login</tabstop>
  <tabstop>btn_close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
